aspnetcore/fundamentals/static-files.md

```diff
@@ -136,7 +136,7 @@ The preceding code allows directory browsing of the *wwwroot/images* folder usin
 
 ## Serve default documents
 
-Setting a default page provides visitors a starting point on a site. To serve a default page from `wwwroot` without a fully qualified URI, call the <xref:Microsoft.AspNetCore.Builder.DefaultFilesExtensions.UseDefaultFiles%2A> method:
+Setting a default page provides visitors a starting point on a site. To serve a default page from `wwwroot` without an [absolute path](https://datatracker.ietf.org/doc/html/rfc3986#section-4.2) or [absolute URI](https://datatracker.ietf.org/doc/html/rfc3986#section-4.3), call the <xref:Microsoft.AspNetCore.Builder.DefaultFilesExtensions.UseDefaultFiles%2A> method:
```

Comment:
I think there might be a simpler way to explain the functionality here, without worrying about absolute URIs, URLs, etc:

```suggestion
Setting a default page provides visitors a starting point on a site. To serve a default file from `wwwroot` without requiring the request URL to include the file's name, call the <xref:Microsoft.AspNetCore.Builder.DefaultFilesExtensions.UseDefaultFiles%2A> method:
```

What do you think? A fully qualified URL, absolute URI, etc, suggests the scheme, domain, etc. An absolute path fits better, but it's still relative to `wwwroot` on the filesystem, which is what's being used here.

If you do approve of this suggestion, there's this a bit further down:

> The first file found from the list is served as though the request were the fully qualified URI.

Which could become something like this?:

    The first file found from the list is served as though the request included the file's name.